android-static-binding-generator/project/build.gradle

```diff
@@ -113,10 +122,17 @@ traverseDirectory = { dir, traverseExplicitly ->
 	}
 
 	currentDir.eachFile(FileType.FILES) { File f ->
-		def file = f.getAbsolutePath();
-		if (file.substring(file.length() - 3, file.length()).equals(".js")) {
-			logger.info("Task: traverseDirectory: Visiting JavaScript file: " + f.getName())
-			inputJsFiles.add(f.getAbsolutePath())
+		def currFile = f.getAbsolutePath();
+		if (currFile.substring(currFile.length() - 3, currFile.length()).equals(".js")) {
+			// Read __worker-chunks.json file containing a list of webpacked workers
+			// ignore worker scripts, so as to not attempt to generate bindings for them
+			if (filterWorkerFiles) {
```

Comment:
We can skip this check. If we don't have a `__worker-chunks.json` file, the `webpackWorkersExcludesList` will be empty.